lib/drivers/insulet/insuletDriver.js

```diff
@@ -1474,9 +1474,13 @@ module.exports = function (config) {
       .with_bgTarget(bgTargetSettings)
       .with_bolus({
         calculator: {
-          enabled: data.eeprom_settings.BOL_CALCS ? true : false
+          enabled: data.eeprom_settings.BOL_CALCS ? true : false,
+          insulin: {
+            duration: data.eeprom_settings.INSULIN_DURATION * 30,
+            units: 'minutes'
```

Comment:
Shouldn't this be `hours`? The validation on the platform side is `0.0 <= x <= 10.0`